src/sqlfluff/core/templaters/jinja.py

```diff
@@ -401,19 +401,12 @@ def slice_file(
         self, raw_str: str, templated_str: str, config=None, **kwargs
     ) -> Tuple[List[RawFileSlice], List[TemplatedFileSlice], str]:
         """Slice the file to determine regions where we can fix."""
-        # The JinjaTracer slicing algorithm is more robust, but it requires
-        # us to create and render a second template (not raw_str) and is only
-        # enabled if the caller passes a make_template() function.
-        make_template = kwargs.pop("make_template", None)
-        if make_template is None:
-            # make_template() was not provided. Use the base class
-            # implementation instead.
-            return super().slice_file(raw_str, templated_str, config, **kwargs)
```

Comment:
I had added this old fallback code in the early days of `JinjaTracer` when we weren't fully converted to using it. Removing it now to ensure that the `templated_str` variable (which may be incorrect due to dbt/Jinja newline stripping) is not used by `slice_file()`, ever.